includes/class-friends.php

```diff
@@ -662,14 +662,23 @@ public static function deactivate_plugin() {
 		wp_unschedule_event( $timestamp, 'cron_friends_refresh_feeds' );
 	}
 
+	/**
+ 	 * Get the required capability for the menu entries.
+   	 */
 	public static function required_menu_role() {
 		return 'edit_private_posts';
 	}
 
+	/**
+ 	 * Check if the user hast the required priviliges.
+   	 */
 	public static function has_required_privileges() {
-		return self::is_main_user() || current_user_can( 'administrator' );
+		return self::is_main_user() || current_user_can( 'edit_options' );
```

Comment:
I think it should be this!
```suggestion
		return self::is_main_user() || current_user_can( 'manage_options' );
```